docs/android-startup-scenarios.md

```diff
@@ -0,0 +1,35 @@
+
+# Android Startup Test
+
+## Prereqs
+
+- Ensure `dotnet` is installed and available with the `dotnet` command for easy xharness installation. Any supported .NET Core version should work. [Dotnet Download](https://dotnet.microsoft.com/en-us/download) or [Daily Dotnet Download](https://github.com/dotnet/sdk/blob/main/documentation/package-table.md)
+- Ensure `xharness` is installed and available with the `xharness` command. The current version in use can be found in the `eng/performance/maui_scenarios_android.proj` file at line 7 (under the tag `MicrosoftDotNetXHarnessCLIVersion`), although any recent version should work. [XHarness Install Instructions](https://github.com/dotnet/xharness?tab=readme-ov-file#installation-and-usage).
+- Have an Android app APK available for testing.
+- Have an Android Device (with developer mode enabled) or emulator connected to computer, and viewable with `xharness android device` or `xharness android adb -- devices -l`.
+
+## Steps
+
+1. Initialize the environment (note the . for bash):
+
+    ```sh
+    cd src/scenarios
+    . ./init.sh  # or `.\init.ps1` on Windows
+    ```
+
+2. Navigate to the `helloandroid` scenario directory:
+
+    ```sh
+    cd helloandroid
+    ```
+
+3. Copy the APK into the `helloandroid` directory.
+4. Run the test:
+
+    ```sh
+    python test.py devicestartup --device-type android --package-path <path to apk (e.g. .)>/<apkname>.apk --package-name <apk package name> [--disable-animations]
+    ```
+
```

Comment:
Maybe add information about where the results of the performance test can be found or an example of what the output should look like.

Reply:
Added section on this.